src/VirtoCommerce.Platform.Web/appsettings.json

```diff
@@ -136,6 +136,7 @@
             "PublicUrl": "http://localhost:10645/cms-content/"
         },
         "AzureBlobStorage": {
+            "RootPath": "cms",
```

Comment:
Extra line

Reply:
Move this line to the end of the "AzureBlobStorage" block.
``` json
"AzureBlobStorage": {
     "ConnectionString": "",
     "CdnUrl": "",
     "RootPath": "cms"
}
```